### DISCUSSION TITLE
How to use synthesizer on a large texts?

### DISCUSSION THREAD

**Discussion by puyebu** [coqui-ai/TTS#917] [General Q&A] [answered]
I am using TTS on windows and the terminal has a limit on how much text I can put in.
Is it possible to target a text file or how do I syntesize large text (multiline, more than 10k characters)?

**Comment by erogol** [comment]
each sentence has 500 character limit. You can split things with `.`. to reduce the size. 

Or you need to find the downloaded model and change the `max_decoder_steps` in the config.json file

**Reply by puyebu** [reply]
I may have been unclear.
Do I have to split the text apart to create multiple output files and then merge them in a single one manually or is it possible to use one big string or text file (separated with dots and/or newlines) to create a single file automatically?

**Comment by WeberJulian** [comment, marked as answer]
I'm copying what I said on the TTS channel:

You can try a small bash/python script. Split the text by '\n' (or what fits your data) and run the TTS command for each split. You can then join the audios together with https://trac.ffmpeg.org/wiki/Concatenate
